### PR TITLE
chore: set import order for spotless plugin

### PR DESCRIPTION
**Pull request by mcollovati**
Imports will be sorted in the following order:

jakarta|java|javax

all dependency imports

com.github.mcollovati

all static imports

### CODE REVIEW

**Comment by mcollovati** [comment]
The same order may be configured in IDE settings.
Instruction should be added in a contribution guide.

**Comment by mcollovati** [comment]
If the order is ok, this PR can be merged even if the formatting check fails, as it uses rules from the target branch.

**Comment by mcollovati** [comment]
A cherry-pick to 1.0 should be performed manually

**Comment by Dudeplayz** [comment]
And whats the reason to pick this order? Personal preference or better visibility etc?

File: pom.xml [hunk 1]
```diff
@@ -227,6 +227,9 @@
                         <palantirJavaFormat>
                             <style>PALANTIR</style>
                         </palantirJavaFormat>
+                        <importOrder>
+                            <order>jakarta|java|javax,,com.github.mcollovati,\#</order>
```
[the remark this answers is not in the document]
Comment (mcollovati) [reply pom.xml:231]: The double.,, means all other imports. # stands for static imports

**Comment by mcollovati** [comment]
> And whats the reason to pick this order? Personal preference or better visibility etc?

For sure personal preferences, but the idea is to have JDK platform imports first, and the imports of our own code near to the class definition, so they are easier to find.

**Review by Dudeplayz — APPROVED** [review]

**Pull request merged**